packages/xml-views-validation/src/utils/deprecated-message-builder.ts

```diff
@@ -27,9 +28,13 @@ export function buildDeprecatedIssueMessage({
       kind = "class";
       name = ui5NodeToFQN(symbol);
       break;
+    case "UI5Aggregation":
```

Comment:
instead of adding more and more cases over time
we can consider trimming the `UI5` prefix and applying "toLowerCase" on the rest.

Reply:
I think there should be a separation between the technical strings and strings that reach the user

Reply:
o.k